code/modules/client/preference/preferences_toggles.dm

```diff
@@ -364,3 +364,11 @@
 	prefs.toggles2 ^= PREFTOGGLE_2_SEE_ITEM_OUTLINES
 	prefs.save_preferences(src)
 	to_chat(usr, "You will [(prefs.toggles2 & PREFTOGGLE_2_SEE_ITEM_OUTLINES) ? "now" : "no longer"] see item outlines on hover.")
+
+/mob/verb/toggle_anonsay()
```

Comment:
might be worth a rename? bit of a nitpick.

Reply:
Could do